src/manager.cpp

```diff
@@ -160,13 +177,122 @@ void Manager::deleteSingleFruVpd(
     logging::logMessage(std::string(i_dbusObjPath));
 }
 
+bool Manager::isValidLocationCode(const std::string i_locationCode)
+{
+    if ((i_locationCode.length() < constants::UNEXP_LOCATION_CODE_MIN_LENGTH) ||
```

Comment:
The condition will give "Umtsp" as valid location code.

Reply:
Updated